frontend/src/app/document-view/document-view.component.html

```diff
@@ -41,7 +41,7 @@
             </div>
             <div *ngIf="tabIndex==0">
                 <ng-container *ngFor="let field of contentFields">
-                    <div class="content" *ngIf="document.fieldValues[field.name]" [innerHtml]="document.fieldValues[field.name] | highlight:query"></div>
+                    <div class="content" *ngIf="document.fieldValues[field.name]" [innerHtml]="highlightedInnerHtml(field)"></div>
```

Comment:
Is there a reason why you did not adapt the highlight pipe for this? If so, I think the pipe is unused now and can be deleted.

Reply:
The highlight pipe is still used as a fallback option if es provides no highlight options (such as for the legacy corpora), as well as for the manual.